src/__tests__/lib/containers/SynapseTableUtils.test.ts

```diff
@@ -1,94 +1,10 @@
 import {
   getColumnIndiciesWithType,
-  getCountFunctionColumnIndices,
-  getSqlUnderlyingDataForRow,
   getUniqueEntities,
 } from '../../../lib/containers/table/SynapseTableUtils'
-import {
-  ColumnType,
-  QueryResultBundle,
-  Row,
-} from '../../../lib/utils/synapseTypes'
-import queryResultBundleJson from '../../../mocks/query/syn16787123.json'
-
-const queryResultBundle: QueryResultBundle =
-  queryResultBundleJson as QueryResultBundle
+import { ColumnType, QueryResultBundle } from '../../../lib/utils/synapseTypes'
 
 describe('Synapse Table Utilities tests', () => {
-  describe('PORTALS-527: get count function column indexes', () => {
```

Comment:
The feature to link a COUNT column to the disaggregated results is currently unused and known to be buggy. The current implementation requires the `sql-parser` package, so I removed it.